integration-libs/opf/payment/root/model/opf-payment.model.ts

```diff
@@ -69,11 +69,11 @@ export interface OpfPaymentSubmitRequest {
 export interface OpfPaymentSubmitInput {
   additionalData: Array<OpfKeyValueMap>;
   paymentSessionId?: string;
-  callbackArray: [
-    OpfPaymentMerchantCallback,
-    OpfPaymentMerchantCallback,
-    OpfPaymentMerchantCallback,
-  ];
+  callbackArray: {
```

Comment:
def more comprehensive.
Good thing, we are not changing the signature of Global function, still some doc part to modify in 'type definitions '
https://help.sap.com/docs/SAP_COMMERCE_CLOUD_PUBLIC_CLOUD/0996ba68e5794b8ab51db8d25d4c9f8a/1fba7ed786b44700a21435ddc07bbc98.html?state=DRAFT&q=global+function
I can make it after we merge